Tests/CSharp/StatementTests/StatementTests.cs

```diff
@@ -83,8 +84,8 @@ internal partial class TestClass
 {
     private void TestMethod(string v)
     {
-        MyEnum b = (MyEnum)Enum.Parse(typeof(MyEnum), v);
-        b = (MyEnum)Enum.Parse(typeof(MyEnum), v);
+        MyEnum b = (MyEnum)Conversions.ToInteger(Enum.Parse(typeof(MyEnum), v));
```

Comment:
Actually that's how decompiled code [looks](https://sharplab.io/#v2:DYLgbgRgNAJiDUAfAkgWwA4HsBOAXAzgAQDKAnvrgKaoCwAUAAoCuEwAlgMaEDCwAhviLd6AUQB2TVIQCypcZPqElhAILTqEStlFiYhebTr1eAogBVKFE4MXKG2NmD5USLQhYrrcAC0wwAFGCEAiS4DmIA5gCUtsrKACJsUhCqRLIGhAC8MnISqAB0DHzY+JT+AOKUuGak6GXpeVFQhGAxdHFxKdkNkoXFpRVVNXX+PahNLW1x4nrELDp61vixhAs8/IJAA=).
I'm almost sure that ```Conversions.ToInteger``` is redundant here. Once this is merged I will create separate task to handle that case and omit ToInteger call.